README.md

```diff
@@ -131,6 +125,37 @@ PROS (+)
 CONS (-)
   * Problem: Managed Service Constraints. You trade control for convenience. You have limited access to the underlying OS and cannot fine-tune every low-level database or system parameter. You are also dependent on AWS's timeline for support of new database versions and features.
 ```
+
+## Security
+### Cognito:
+```
+
+PROS (+) 
+  * Easy to setup SMS MFA: Phone verification is provided out of the box, the web page and verification logic is provided by it.
+  * Great support for Mobile Apps: There are Android and iOS SDKs available
+  * Fully managed by AWS, cloud-based
+CONS (+)
+  * User pool in AWS Cognito is not multi-tenant: In order to create a multi-tenant system we have to have a dedicated user pool per tenant.
+  * Limitation of 1000 user pools per AWS account: very little if we want to scale a multi-tenant application.
+  * Limited Customization
+  * Tight AWS ecosystem integration (it might not apply to our case since we are all AWS)
+```
+
+### Keycloak:
+```
+
+PROS (+) 
+  * Easily configurable to multi-tenant
+  * Open-source, self hosted
+  * Cost: Free (Infrastructure costs apply)
+  * Highly customizable
```

Comment:
@Thorugoh Could you update all PROS/CONS following this approach

E.g Customization: You can customize how the login screen, email templates, and account management UIs look